lib/orderbook/OrderBook.ts

```diff
@@ -387,6 +391,15 @@ class OrderBook extends EventEmitter {
     return true;
   }
 
+  private persistTrade = async (quantity: number, makerOrder: orders.PeerOrder | orders.OwnOrder, takerOrder: orders.OwnOrder | orders.PeerOrder) => {
+    await this.repository.addOrdersIfNotExists([makerOrder, takerOrder]);
```

Comment:
Looking at the previous comments/changes I also think it would be cleaner to just have:
```
await this.repository.addOrderIfNotExists(makerOrder);
await this.repository.addOrderIfNotExists(takerOrder);
```
instead of `this.repository.addOrdersIfNotExists`.

Reply:
I'd be OK with keeping it, either way I think it would be nice to `await` them in parallel using `Promise.all` (which is currently happening in the `addOrdersIfNotExists` method.

Unless there's another time we'd need to persist multiple orders at once (I can't think of any currently), my preferred approach solution in one line solution and without needing an additional method would be:
```javascript
await Promise.all([this.repository.addOrderIfNotExists(makerOrder), this.repository.addOrderIfNotExists(takerOrder)]);
```